playbooks/os_updates.yml

```diff
@@ -0,0 +1,29 @@
+---
+- hosts: all
+  serial: 2
+  remote_user: pulsys
+  become: true
+  pre_tasks:
+
+    - name: test for python on new VM
```

Comment:
We don't need the pre-tasks for an update playbook. Remove these, we will add them in the "bare-VM" playbook (TBD).